mllib-local/pom.xml

```diff
@@ -56,6 +56,11 @@
       <groupId>org.apache.spark</groupId>
       <artifactId>spark-tags_${scala.binary.version}</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.spark</groupId>
+      <artifactId>spark-core_${scala.binary.version}</artifactId>
```

Comment:
We won't be able to include this dependency - mllib-local needs to be independent of core.

Reply:
That is needed for two bits:
 - `Utils.classForName`: a scalastyle rule indicated I should do that. I can always disable the rule locally and use `Class.forName` directly.
 - `Benchmark` and `BenchmarkBase`: I followed how `org.apache.spark.mllib.linalg.UDTSerializationBenchmark` does it. Since that is only for testing, is it an issue, or could that stay as-is?

Reply:
I know, but we won't be able to allow that dependency. You can 'inline' Utils.classForName, and eventually we won't need the benchmark (or maybe the test-scope dependency is OK). For experiments now it's fine.